### ISSUE TITLE
Remove "data available soon" when spatial data becomes available on the portal

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/scpca-docs#63]
Currently, we have a line under the processing section for spatial data that states `data available soon`. After we make spatial data available we will want to make sure we remove this line.

**Comment by davidsmejia** [comment]
Do we just want to remove the whole section?

```

### Combining CITE counts with RNA counts

The unfiltered CITE-seq and RNA-seq count matrices often... [truncated]

## Spatial Transcriptomics

**_Data Available Soon_**

### Mapping and quantification using Space Ranger

Processing spatial transcriptomics libraries requires... [truncated]


```

**Comment by allyhawkins** [comment]
Which section are you referring to? The `Mapping and quantification using Space Ranger` is specifically referring to spatial transcriptomics libraries so I would want to make sure that is under the `Spatial Transcriptomics` header. We could remove that entire section about processing spatial data but we also have information on the download page about spatial data so I think it would be more work to remove it at this point rather than keep it and then remove the `Data Available Soon` line when it's ready.

**Comment by jaclyn-taroni** [comment]
My understanding of this scope of this issue is just to remove the Data Available Soon line when spatial data is available to download from prod.

**Comment by jaclyn-taroni** [comment]
Closed via #81
